src/tests/ftest/util/command_utils.py

```diff
@@ -117,7 +118,7 @@ def _run_process(self):
             # Block until the command is complete or times out
             return run_command(
                 command, self.timeout, self.verbose, self.exit_status_exception,
-                "combined", env=self.env)
+                self.output_check, env=self.env)
```

Comment:
What would be the downside to just removing the "combined" setting here?  It seems that until recently the version of avocado it never worked so why not configure the new version to behave as the old version did?

Reply:
The default is now "both".